biorbd_optim/acados_interface.py

```diff
@@ -240,8 +233,8 @@ def get_optimized_value(self, ocp):
 
         return out
 
-    def solve(self):
-        from acados_template import AcadosOcpSolver
-
+    def solve(self, ocp):
```

Comment:
I think it would make more sense to keep a reference to ocp in self while __init__ is called so one doesn't have to know that when they changed something they need to pass ocp